src/kerberos_auth_server.py

```diff
@@ -307,13 +301,13 @@ def handle_client_request(self, request):
             print("handle_client_request error: \n" + str(e))
             exit(1)
 
+
     def start_server(self):
         """
         infinite loop of listening server
         :return:
         """
         try:
-            print(f"Server Started on port {self.port}")
```

Comment:
don't delete that it is helpful for debugging a lot of network errors and nice to show in video